data/genomics/sarscov2/genome/proteome.fasta

```diff
@@ -0,0 +1,255 @@
+>ENSSASP00005000002.1 pep primary_assembly:ASM985889v3:MN908947.3:266:21555:1 gene:ENSSASG00005000002.1 transcript:ENSSAST00005000002.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:ORF1ab description:ORF1a polyprotein;ORF1ab polyprotein [Source:NCBI gene (formerly Entrezgene);Acc:43740578]
+MESLVPGFNEKTHVQLSLPVLQVRDVLVRGFGDSVEEVLSEARQHLKDGTCGLVEVEKGV
```

Comment:
This file was already added in https://github.com/nf-core/test-datasets/pull/277 so I am a bit confused as to how you pulled in these changes via `git`?

Reply:
Oh, yes, that's my bad probably. The thing is that I did `git pull upstream modulles` to have the latest version of nf-core's `modules` branch before pushing that commit. But probably I should not have done that, because I need to add only my own files! So, this peptide file was already merged into `modules` and hence made its way into my PR. What should I do with this?

Reply:
Let's find out 🥁